libraries/protocol/fee_schedule_calc.cpp

```diff
@@ -53,6 +53,18 @@ namespace graphene { namespace protocol {
       }
    };
 
+   /**
+    * Specialization for htlc_create_operation
+    * 
+    * For HTLCs, after HF_BSIP64, we need to add in the fee per kb
+    */
+   template<>
+   uint64_t calc_fee_visitor::operator()(const htlc_create_operation& op)const
+   {
+      transfer_operation::fee_parameters_type t;
```

Comment:
If we decided to go this approach, we need to pass in the transfer kb fee of current fee schedule, but not use the default one.